### ISSUE TITLE
"Report Status" UI oversimplifies the state of Test Plan Runs

### ISSUE THREAD

**Issue by jugglinmike** [w3c/aria-at-app#1046]
Today, given a Test Plan Run with 3 tests and 6 assertions (where the NVDA Bot has collected all 3 AT responses and assigned 0 verdicts), the "Report Status" dialog will report "100% complete by NVDA Bot with 0 conflicts." This is misleading because the verdicts are absent, so they cannot be in conflict or in alignment with those in other Test Plan Runs.

Here are some potential solutions:

- Treat unassigned verdicts as inherently conflicting. In the situation described above, the status text would instead read, "100% complete by NVDA Bot with 6 conflicts"
- Extend the status text to fully document the relevant state. In the situation described above, the status text would instead read, "100% complete by NVDA Bot with 0 conflicts and 6 missing verdicts"

**Comment by ccanash** [comment]
Hi @mcking65  @mfairchild365, do you agree this is an issue and do you have a preference for either of these solutions or a third option?
